app/views/DR/Home.js

```diff
@@ -0,0 +1,219 @@
+import React, { Component } from 'react';
+import {
+  View,
+  TouchableOpacity,
+  Image,
+  StyleSheet,
+  Text,
+  Card,
+} from 'react-native';
+import { Left } from 'native-base';
+import {
+  widthPercentageToDP as wp
+} from 'react-native-responsive-screen';
+import 'moment/locale/es';
+import moment from 'moment';
+//import { Notifications } from 'expo';
+//import { Feels, Aurora } from '../../DR/components/ActionCards'; // <LastBulletin navigation={navigation} />
+import { requestManager } from '../../services/DR/requestManager';
+import styles from '../../components/DR/styles/styles';
+import Colors from '../../constants/DR/colors';
+
+export default class HomeScreen extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cases: 0,
+      deaths: 0,
+      recovered: 0,
+      todayCases: 0,
+      refreshing: false,
+    };
+  }
+
+  getCases = async () => {
+    try {
+      const {
+        cases,
+        deaths,
+        recovered,
+        todayCases,
+      } = await await requestManager.getAllCases();
+
+      this.setState({
+        cases,
+        deaths,
+        recovered,
+        todayCases,
+        refreshing: false,
+      });
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  refresh = () => {
+    this.setState({ refreshing: true });
+    const cases = this.getCases();
+    console.log(cases);
+  };
+
+  async componentDidMount() {
+    const { navigation } = this.props;
+    navigation.setOptions({ headerShown: false });
+    const cases = this.getCases();
+    console.log(cases);
+    this._notificationSubscription = Notifications.addListener(
+      this._handleNotification
+    );
+  }
+
+  getSettings() {
+    return (
+      <TouchableOpacity
+        style={stylesSavePath.settingsContainer}
+        onPress={() => {
+          this.props.navigation.navigate('SettingsScreen');
+        }}>
+      </TouchableOpacity>
+    );
+  }
+
+  _handleNotification = (notification) => {
+    const { navigation } = this.props;
+    const { data } = notification;
+
+    if (data) {
+      const { type } = data;
+      if (type == 'bulletin') {
+        navigation.navigate('App', {
```

Comment:
Could we confirm that exists? I believe it should be `BulletinsScreen`

Reply:
nope I took it off for now